packages/search-ui/src/__tests__/SearchDriver.test.js

```diff
@@ -89,7 +89,7 @@ it("will default facets to {} in state if facets is missing from the response",
     }
   });
 
-  expect(doesStateHaveResponseData(stateAfterCreation)).toBe(true);
+  //expect(doesStateHaveResponseData(stateAfterCreation)).toBe(true);
```

Comment:
FYI, looks like this was a comment accidentally left in. Should it be uncommented or entirely removed?

Reply:
@constancecchen Whoops, I will take care of that.

Reply:
It's easy to get your PRs to go green by commenting out test lines.